docs/src/lib/indexing.md

```diff
@@ -62,6 +62,7 @@ a `SubDataFrame` or a `DataFrameRow` always returns a `DataFrameRow` (which is a
                      the same as `df[!, col][rows]`;
 * `df[rows, cols]` -> a `DataFrame` containing copies of columns `cols` with only the entries corresponding to `rows` selected;
 * `df[!, col]` -> the vector contained in column `col` returned without copying; the same as `df.col` if `col` is a valid identifier.
+* `df[!, cols]` -> the same as `select(df, cols, copycols=false)`.
```

Comment:
Also describe what this does? Indexing is more fundamental than `select`.

Reply:
Added here and below. Thank you.